echo-sql/README.md

```diff
@@ -0,0 +1,120 @@
+# Example URL Shortener
+
+A sample url shortener app to test Keploy integration capabilities using [Echo](https://echo.labstack.com/) and PostgreSQL. 
+
+## Installation
+
+### Start keploy server
+
+```shell
+git clone https://github.com/keploy/keploy.git && cd keploy
+docker-compose up
+```
+
+### Setup URL shortener
+
+```bash
+git clone https://github.com/keploy/example-url-shortener && cd example-url-shortener/echo-sql
```

Comment:
please update the URL, since it's changed now. We also need to update this for the other gin-mongo example.

Reply:
Fixed in most recent commit.